src/hooks/historikk/useDialogMedBehandlerHistorikk.ts

```diff
@@ -0,0 +1,59 @@
+import { HistorikkEvent } from "@/data/historikk/types/historikkTypes";
+import { useBehandlerdialogQuery } from "@/data/behandlerdialog/behandlerdialogQueryHooks";
+import {
+  MeldingDTO,
+  MeldingResponseDTO,
+} from "@/data/behandlerdialog/behandlerdialogTypes";
+import { meldingTypeTexts } from "@/data/behandlerdialog/behandlerdialogTexts";
+
+interface DialogMedBehandlerHistorikk {
+  isDialogMedBehandlerHistorikkLoading: boolean;
+  isDialogMedBehandlerHistorikkError: boolean;
+  dialogMedBehandlerHistorikk: HistorikkEvent[];
+}
+
+function beregnAvsender(dialogmelding: MeldingDTO): string {
```

Comment:
Usikker på hva jeg tenker om `beregn…` begrepet her 🤔 Kan det holde med noe i denne duren? 
```suggestion
function avsenderText(dialogmelding: MeldingDTO): string {
```
Evt. bare `function avsender(…)`

Reply:
Er vel tildels enig at beregn kan vel teknisk sett ha andre implikasjoner. Forsøkte å si noe litt mer enn det jeg synes `avsenderText(..)` og evt. `avsender(..)` ville sagt, men endrer det til forslaget du kom med ettersom det er godt nok.